### DISCUSSION TITLE
When can we expect the next release of KaTeX?

### DISCUSSION THREAD

**Discussion by Awbmilne** [KaTeX/KaTeX#2525] [General] [answered]
Hello!

Started up a new semester at University and Im using [Joplin](https://joplinapp.org/) to take my notes in Markdown.
As you may expect, Joplin uses KaTeX in the background to render Math function. 
So **THANK YOU!** to all the KaTeX maintainers.

During this semester, we are working more with AC circuits, and as a result, we are dealing with a lot of Phasors.
Its currently not supported in Joplin (v0.12.0 of KaTeX), so I took a snoop and found #2406 with 42cc5e507e9300ad03969a50299c0458b7862056.

Thank you to  @ ronkok  @ kevinbarabash for adding `\phase` (purposefully not @-ing them)

When can we expect the next release of KaTeX?\
 Im really looking forward to using phasors in my notes!

Thanks!

**Comment by VergeDX** [comment]
I also want #2488, and use this feature in Notion.
When can we expect the next release of KaTeX?

**Comment by k4b7** [comment, marked as answer]
I've just publish [v0.13.0](https://github.com/KaTeX/KaTeX/releases/tag/v0.13.0).
